node_modules/app/format_date.js

```diff
@@ -6,12 +6,16 @@ module.exports = function (d) {
   return date[1]
 }
 
+module.exports.date_string = function (d) {
```

Comment:
Hmm. How about

```js

module.exports.date_only = date_only

function date_only (d) {
  return d.toISOString().split('T')[0]
}

function format_date (d) {
  ...
  var dateString = date_only(date)
  return ...
}
```

We'd then pass the date in the query to `new Date` ourselves first.

Only concern is the fact this could blow up ...